pkg/spec/cluster_tls.go

```diff
@@ -22,8 +22,9 @@ type TLSPolicy struct {
 }
 
 type StaticTLS struct {
-	// ServerSecretName contains peer-interface and client-interface server x509 key/cert, along with peer and client CA cert.
+	// ServerSecretName contains peer-interface and client-interface server x509 key/cert and CA cert.
```

Comment:
@xiang90 

I think this PR is for peer TLS. It also help cleaned up the spec a little. But it doesn't change original purpose.
If you have questions on the purpose of TLS spec on client part, we should resolve it in another issue.